arm-recoveryservices/2016-06-01/swagger/vaults.json

```diff
@@ -615,8 +684,7 @@
       ],
       "properties": {
         "properties": {
-          "$ref": "#/definitions/VaultProperties",
-          "x-ms-client-flatten": true
+          "$ref": "#/definitions/VaultProperties"

```

Comment:
@olydis do we have any recommendations here, the nesting of properties seems a little unnatural.